section 4_Flask_Restfull/user.py

```diff
@@ -0,0 +1,7 @@
+
+
+class User:
+    def __init__(self,_id,username,password) -> None:
```

Comment:
why _id as arg?

Reply:
The author used it. but in the next section, he will improve it and generate it automatically.